bib/cmd/bootc-image-builder/image.go

```diff
@@ -226,16 +228,23 @@ func manifestForISO(c *ManifestConfig, rng *rand.Rand) (*manifest.Manifest, erro
 		customizations = c.Config.Customizations
 	}
 
-	img.Users = users.UsersFromBP(customizations.GetUsers())
-	img.Groups = users.GroupsFromBP(customizations.GetGroups())
+	img.Kickstart = &kickstart.Options{
```

Comment:
I think we should consider this and think about it a bit, but either way it would have to go in `images`.

If we generalise it though, I think it'll be a bit conditional on the image type.  I'm thinking the image installer will have its kernel args in the image itself and wont need to have them set in the kickstart.  Or maybe we can make it a general rule that for installers, we always set it in the kickstart.